pages/global-profile/views/index.jsx

```diff
@@ -33,6 +34,29 @@ const selector = ({
 
 const formatDate = (date, format) => (date ? dateFormatter(date, format) : '-');
 
+const ProfileMerges = ({ profile }) => {
+  if (profile.profileMerges.length === 0) {
```

Comment:
Just in case it's not there at all.

```suggestion
  if (!profile.profileMerges || profile.profileMerges.length === 0) {
```